docs/openhabian.md

```diff
@@ -478,7 +478,7 @@ If you are not able to access your system via the openHAB dashboard or SSH after
 <a id="switch-openhab-branch"></a>
 #### Can I switch openHAB 2 and 3 via openHABian branches ?
 openHABian installs the latest stable build of openHAB.
-The standard openHABian `stable` and `master` branches will install openHAB version 2 and the `openHAB3` branch will install the new openHAB version 3.
+The standard openHABian `openHAB3` and `main` branches will install the new openHAB version 3 and the old `stable` and `master` branches will install the old openHAB version 2.
```

Comment:
that would then only be 'stable' no more 'master', right ?

Reply:
Hmm, I never thought about that. I think we keep master for now and remove it after a little while so we dont break anyone's repositories at that time I will remove it.

Reply:
Ok. I think you should add an entry to the NEWS page explaining the new branch structure, asking users of master to migrate over to 'stable'.

Reply:
Will do later today. So far the transition seems to have gone smoothly.